scripts/st2bootstrap-deb.sh

```diff
@@ -653,8 +653,26 @@ EOT"
 }
 
 install_st2chatops() {
-  # Add NodeJS 6 repo
-  curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+  INSTALLABLE_ST2CHATOPS_VERSION=$(apt show st2chatops | grep 'Version:' | sed 's/^Version: \([[:digit:]]\+\)\.\([[:digit:]]\).*/\1.\2')
+
+  # Check if the st2chatops version is >= 3.0
+  python -c <<EOF
+import sys
+if '${INSTALLABLE_ST2CHATOPS_VERSION}' >= '3.0':
```

Comment:
I don't think this will work as expected.

We either need to parse it into a tuple or use `StrictVersion` from distutils or similar.

Reply:
I don't think we need to do this at all. The only time you would want to install older node version is if you're passing flags to the install script to tell it to install an older ST2 version. 

But in that case it will retrieve the OS-specific install script from the relevant branch - see https://github.com/StackStorm/st2-packages/blob/master/scripts/st2_bootstrap.sh#L170

So we don't need to put this code in here. Anyone pulling this code will be installing from dev, or in future it will be >= 3.x